.swiftlint.yml

```diff
@@ -0,0 +1,6 @@
+included:
+  - Sources
+  - Tests
+disabled_rules:
+  - function_body_length
```

Comment:
@Palleas do you know how to disable this rule only for Tests?

Reply:
I don't think you can, one solution would be to have exceptions for specific files, maybe?